app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -930,16 +930,6 @@ public HomeFragment createHomeScreen() {
         return fragment;
     }
 
-    @Override
```

Comment:
Why are we removing these two telemetry pings?

Reply:
They are not used by the data analyst.